client/cmd/project/register.go

```diff
@@ -38,34 +39,42 @@ func NewRegisterCommand() *cobra.Command {
 		Use:     "register",
 		Short:   "Register project if it does not exist and update if it does",
 		Example: "optimus project register [--flag]",
+		PreRunE: register.PreRunE,
 		RunE:    register.RunE,
 	}
+	// Config filepath flag
```

Comment:
let's start reducing these kind of comments. while in some other place these also exist, let's start reducing it.